docs/HowTo/Deploy/Lite-Block-Explorer.md

```diff
@@ -1,101 +1,107 @@
 description: Ethereum Lite Explorer
 <!--- END of page meta data -->
 
-Use the [Alethio Ethereum Lite Explorer](https://lite-explorer.aleth.io/) to explore blockchain data 
-at the block, transaction, and account level.
- 
-The Alethio Ethereum Lite Explorer is a web application that connects to any Ethereum 
-JSON-RPC enabled node. No online server, hosting, or trusting third parties to display the blockchain
-data is required. 
+Use the [Alethio Ethereum Lite Explorer](https://lite-explorer.aleth.io/) to explore blockchain
+data at the block, transaction, and account level.
+
+The Alethio Ethereum Lite Explorer is a web application that connects to any Ethereum
+JSON-RPC-enabled node. It does not require an online server, hosting, or trusting third parties to
```

Comment:
Same comment about `it` as about `this` on another PR.  Generally, we try and avoid using `it` because it adds to the cognitive load of the reader.  They have to think about `it` in relation to the previous sentence to understand the meaning.  The difficulty increases for ESL readers and for translation. 

Could be rewritten as: 
The Explorer does not require....

Reply:
Done.